docs/conf.py

```diff
@@ -144,16 +144,45 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 
-html_theme = 'edx_theme'
+html_theme = 'sphinx_book_theme'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
-
+html_theme_options = {
+    "repository_url": "https://github.com/openedx/django-user-tasks",
+    "repository_branch": "master",
+    "path_to_docs": "docs/",
+    "home_page_in_toc": True,
```

Comment:
[question] Why are we leaving out `logo_only` ?

Reply:
Support for that seems to have been removed in this PR: https://github.com/executablebooks/sphinx-book-theme/pull/668/files#diff-56e5475bc04460ee05cd5ac9199ebbfa3e21f1bfa64ce5469cfb2f1890b41ccfL42

There is now a different mechanism for this as outlined here: https://github.com/executablebooks/sphinx-book-theme/pull/668/commits/b0e6b524c33e290d406557f2550005cf83934056#diff-ffa6fcfcf96f0b20b242a31c1346439658bb90a4e42e3c8fc8d0b1bc3d4b7277L59

The new default behaviour seems to match `logo_only = True` though.